packages/slate-editor/src/extensions/story-embed/types.ts

```diff
@@ -1,22 +1,15 @@
 import type { StoryEmbedNode } from '@prezly/slate-types';
-import type { ReactNode } from 'react';
+import type { FunctionComponent } from 'react';
```

Comment:
nit: you can use `ComponentType` just in case of classic class components

Reply:
Yeah, I've tried that. See 5bc526dc3a6be5fb69aca53cffe9fa51c97678fd :)

Reply:
hm, if it is not a component i would avoid using *Component type, just plain callback function

Reply:
Done in 54a46704fba2b7a4c4dde5323fee5284e9f7a3c6. Thanks! :+1: